jx_sqlite/query_table.py

```diff
@@ -278,7 +278,7 @@ def query(self, query):
 
             output = Data(
                 meta={"format": "table"},
-                header=column_names,
+                header=[split_field(h)[0] if "\\" else h for h in column_names],
```

Comment:
split_field is creating list(with only one element) out of variable path, that's why `split_field(h)[0]`.